src/components/NextButton.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <button @click="onClick" :class="getButtonStyle()"><slot></slot></button>
```

Comment:
slot을 사용하여 button의 title을 동적으로 설정해주셨네요 👍
현재 구조에서는 slot을 더 딥하게 써볼 일은 많지 않을거같지만, 각각 다른 슬롯 콘센트를 대상으로 여러 슬롯을 네임 명시를 통해 지정해줄 수 있어요! 복잡한 구조에서 동적으로 여러 슬롯이 필요하다하면 사용해보셔도 좋을거같습니다!

이 부분도 굉장히 마이너하지만, 
하위 태그가 존재할때는 가로 정렬보단 수직으로 정렬하는 것이 더 좋을거같다는 개인적인 생각입니다 ㅎㅎ
이런 작은 부분들이 모여서 더 좋은 가독성을 만들어 준다고 생각해서요!

```suggestion
  <button @click="onClick" :class="getButtonStyle()">
    <slot></slot>
  </button>
```

Reply:
옹... 여러 슬롯을 지정해줄 수 있군요?! React의 children 정도로만 생각했는데..! 관련해서 찾아봐야겠네요.
감사합니다 :)~!